### DISCUSSION TITLE
How to turn off Typescript?

### DISCUSSION THREAD

**Discussion by jamesryan-dev** [vercel/next.js#16651] [Help] [answered]
From the get go the starter I've been using has been configured with Typescript and as my project has developed I've decided against using it.. this was fine at a component level in which I converted pages/comps accordingly however since attempting to do so with _app.tsx and _document.tsx (into _app.js and _document.js) I'm receving errors regards finding the respective pages with their .tsx extension.. 

Cmd+f doesn't seem to resolve where these files are being called and thus why im asking if the community know of a way in which i can turn off typescript?

**Comment by jamesmosier** [comment, marked as answer]
Hi @jimmynames. Did you remove the `tsconfig.json` file from the root folder of your project? Additionally you should also remove `typescript` from your `package.json` and then reinstall your dependencies via `yarn install` or `npm install`

**Reply by jamesryan-dev** [reply]
safe thankyou - I attempted to delete this file in my TE but it kept crashing so i assumed it shouldn't of been deleted however upon doign so and `npm uninstall` -ing it's worked a treat -thankyou!
